src/main/java/net/glowstone/command/CommandUtils.java

```diff
@@ -116,15 +116,19 @@ public static String prettyPrint(String[] strings) {
         return builder.toString();
     }
 
+    private static GlowWorld getDefaultWorld() {
```

Comment:
Although not part of the scope of this PR, I think it would be better to add this method to the `Server` interface (Glowkit) and implement it in the `GlowServer` class. Can you put a `todo` here for this?

Reply:
Done.